### PR TITLE
Promise tracker v155

### PR DESCRIPTION
**Pull request by jell0wed**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
There are no way to keep track of which promises are lingering / pending if one never resolve/reject a promise he/she creates.

**What is the new behavior (if this is a feature change)?**
A promise counter has been implemented to the angular qFactory that counts the number of pending/lingering promises that have not been resolved. The counter gets incremented whenever a new promise is created, and is decremented as promises are resolved or rejected. 

Examples:
https://plnkr.co/edit/ydki6XZkePEfexqaUQYE

### CODE REVIEW

File: src/ng/q.js [hunk 1]
```diff
@@ -257,25 +257,34 @@ function qFactory(nextTick, exceptionHandler) {
...
+    var d = new Deferred(trackPromise);
     //Necessary to support unbound execution :/
     d.resolve = simpleBind(d, d.resolve);
     d.reject = simpleBind(d, d.reject);
     d.notify = simpleBind(d, d.notify);
     return d;
   }
 
-  function Promise() {
+  var countPromises = function() {
```
Comment (eliranek1) [inline src/ng/q.js]: "getPromiseCount"

**Review by eliranek1 — COMMENTED** [review]
Revert changes in yarn-error.log and yarn lock

**Comment by jell0wed** [comment]
@eliranek1 I updated the demo on plunker.

**Comment by jell0wed** [comment]
Updated the promiseTracker to the window.promise object.

**Comment by jell0wed** [comment]
This PR was based on branch v1.5.x which caused error in our build. 

See updated PR based on branch dmxfee-mater #3

**Pull request closed without merging**